src/pyrqlite/cursors.py

```diff
@@ -148,6 +150,10 @@ def _substitute_params(self, operation, parameters):
     def _get_sql_command(self, sql_str):
         return sql_str.split(None, 1)[0].upper()
 
+    def convert_query(self, query):
+        FORMAT_QMARK_REGEX = re.compile(r'(?<!%)%s')
```

Comment:
This isn't safe, the parameter style really needs to be adjustable in the ORM, for example sqlalchemy's SQLiteDialect declares `default_paramstyle = 'qmark'` here:

https://github.com/zzzeek/sqlalchemy/blob/rel_1_2_10/lib/sqlalchemy/dialects/sqlite/base.py#L1043